docs/tutorials/oss-to-enterprise-migration-new.md

```diff
@@ -0,0 +1,99 @@
+# Upgrading from Open Source to Enteprise Using a New Server
+
+Congratulations, you have decided to use Rundeck Enterpise! You are on you way to automating your workflows. Before that happens, you need to get your Rundeck Enterprise set up. Fear not, this guide will get you set up no matter what kind of Rundeck user you are. 
+
+## RPM Upgrade
+
+* Delete job execution history. If you do not, then the exporting of projects will take a much longer time.
+* Export project setting from Rundeck Open Source. Go to the project that you wish to export, go to project settings and select "Export Archive." On the next page, you will be able to choose what all you want exported (Jobs, Executions, Configurations, etc.)
+* To begin, stop your Rundeck OSS with the following command:
+```
+        service rundeckd stop
+```
+* Remove Rundeck using the following command:
+```
+        yum remove rundeck -y
+```
+Following the last command, modified files from `/etc/rundeck` will get .rpmsave extension
+* Now, we are going to install rundeck enterprise by running the following commands:
+ ```
+    curl https://bintray.com/rundeckpro/rpm/rpm | sudo tee /etc/yum.repos.d/bintray-rundeckpro-rpm.repo
+    sudo yum install java rundeckpro-enterprise
+    yum install rundeckpro-enterprise -y
+ ```   
+* New files are in `/etc/rundeck/` among the .rpmsave. Do not restore full old files, just custom settings from .rpmsave to the new ones (except profile file), because enterprise files have new options.
+* Now, need to restore converter passwords (and custom settings) in new rundeck-config-properties
+* Do not edit profile file, add your custom config from it (if any) to /etc/sysconfig/rundeckd (create the file if it is not there)
+* Restore other useful information from other files like ssl.properties or realm.properties.
+* If you are going to use an external database, now is the time to set it up. In order to do so, follow the directions [here](https://docs.rundeck.com/docs/administration/configuration/database/#database-overview)
+* Start your new Rundeck enterprise instance by running the following command:
+```      
+        service rundeckd restart
+```
+* Once in Rundeck, import the project archive files that you created in step 2. Create a new project, go to project settings and select "Import Archive." On the next page, you will be able to choose what all you want imported (Jobs, Executions, Configurations, etc.)
+* Install license by choosing the license key file that was sent to you by a Rundeck team member. 
+* Set execution mode to active
+* Go to cluster manager, select  jobs, and select all orphan jobs and click action button/Change Job Schedule Owner and assign to a cluster member.
+
+## Debian/Ubuntu Upgrade
+
+* Delete job execution history. If you do not, then the exporting of projects will take a much longer time.
```

Comment:
These are repeated.  We might want a section at the top for "Pre-Requisites" or "Prepare your upgrade" so we only have to manage that content once.  DRY principles ;)

Reply:
Updated